src/status_im/ui/screens/wallet/accounts/views.cljs

```diff
@@ -94,17 +96,34 @@
                      :key-fn             :name
                      :render-fn          (render-asset (:code currency) prices-loading?)}]))
 
-(views/defview total-value []
+(views/defview total-value [{:keys [y]}]
   (views/letsubs [currency        [:wallet/currency]
                   portfolio-value [:portfolio-value]
                   prices-loading? [:prices-loading?]]
-    [react/view {:style {:padding-horizontal 16}}
-     [react/view {:style {:flex-direction :row}}
-      (if prices-loading?
-        [react/small-loading-indicator]
-        [react/text {:style {:font-size 32 :color colors/black :font-weight "600"}} portfolio-value])
-      [react/text {:style {:font-size 32 :color colors/gray :font-weight "600"}} (str " " (:code currency))]]
-     [react/text {:style {:color colors/gray}} (i18n/label :t/wallet-total-value)]]))
+    [reanimated/view {:style {:position  :absolute
+                              :left      8
+                              :top       0
+                              :transform [{:translateY
+                                           (reanimated/interpolate y
+                                                                   {:inputRange       [0 styles/scroll-offset]
```

Comment:
@Ferossgp please make sure it doesn't become a sword, please keep it in 80 chars limit. It can be extracted if necessary

Reply:
![Screenshot 2020-04-10 at 11 22 18](https://user-images.githubusercontent.com/2364994/78975815-a2928200-7b1d-11ea-9251-3981254e4e26.png)
I can't read it without horizontal scrolling even here